### ISSUE TITLE
Global ranking incorrectly numbers your score as 1st

### ISSUE THREAD

**Issue by CaptainBasculin** [ppy/osu-stable-issues#352]
![osu!_Qdd2lAWvQ3](https://user-images.githubusercontent.com/48386548/73961522-cf5bca80-491d-11ea-8be5-91f4b2b3c180.png)
(In the screenshot my score is numbered as #1 of 15k, which is not the actual case.)

This happens rarely, and is gone after clicking another map and back. It's a minor thing that doesn't bother me, but shouldn't exist.

**Comment by peppy** [comment]
since when? how often? what is your actual position? what is the beatmap? details? please?

**Comment by CaptainBasculin** [comment]
Sorry for late reply
Since when: I saw this occuring yesterday. Can't remember it existing before.

How often: I played for an hour and a half at that time, this appeared like 3 times on random beatmaps. 

What is the beatmap: (I skipped the first map I got this out of instinct, can't remember which one it was) Marisa wa Taihen [Hard] By DJPop was my secound encounter of this bug, third time I encountered this was Tenshinranman High Collar Hime [Banana's Hard], but I doubt map names are relevant to this, since I got this on two other maps while f2ing. 

What was your position: My actual position was around 5k-6k on this map (Hime), didn't submit a score on it for years.

On the previous map (marisa) I checked it after I got 100ish rank play submitted. It displayed me as rank 1. Picked another diff, repicked the diff and it fixed itself. 

Can't remember the first occurance since I hit the random button key instantly, as stated before.
